creator-node/src/routes/users.js

```diff
@@ -160,30 +162,31 @@ module.exports = function (app) {
 
   /**
    * Returns latest clock value stored in CNodeUsers entry given wallet, or -1 if no entry found
-   * Also returns boolean indicating whether a sync is in progress
-   * Conditionally returns info on total and skipped CIDs for user
+   * Returns boolean indicating whether a sync is in progress
+   * Optionally returns info on total and skipped CIDs for user
+   * Optionally returns user filesHash
    */
   app.get(
     '/users/clock_status/:walletPublicKey',
     handleResponse(async (req, res) => {
       const redisClient = req.app.get('redisClient')
 
-      let walletPublicKey = req.params.walletPublicKey
-      walletPublicKey = walletPublicKey.toLowerCase()
-
-      const returnSkipInfo = !!req.query.returnSkipInfo
+      const walletPublicKey = req.params.walletPublicKey.toLowerCase()
+      const returnSkipInfo = !!req.query.returnSkipInfo // default false
+      const returnFilesHash = !!req.query.returnFilesHash // default false
 
       const response = {}
 
-      async function fetchClockValueFromDb() {
-        // Fetch clock value from DB
-        const cnodeUser = await models.CNodeUser.findOne({
-          where: { walletPublicKey }
-        })
-        const clockValue = cnodeUser ? cnodeUser.dataValues.clock : -1
-        response.clockValue = clockValue
+      const cnodeUser = await models.CNodeUser.findOne({
+        where: { walletPublicKey }
+      })
+      const cnodeUserUUID = cnodeUser
+        ? cnodeUser.dataValues.cnodeUserUUID
+        : null
+      const clockValue = cnodeUser ? cnodeUser.dataValues.clock : -1
+      response.clockValue = clockValue
 
-        // Return CIDSkipInfo if requested
+      async function fetchCIDSkipInfoIfRequested() {
         if (returnSkipInfo && cnodeUser) {
```

Comment:
shouldn't this be cnodeUserUUID?

Reply:
either works given above conditional, but `cnodeUserUUID` is safer here - changed
also this is old unchanged code but yah good call